docs/output.md

```diff
@@ -547,6 +548,21 @@ This directory will hold phyloseq objects for each taxonomy table produced by th
 
 </details>
 
+### VSEARCH cluster
+
+This directory will hold the centroid fasta file, the filtered asv count table (after merging non-centroid counts with their respective centroid counts), and a stats table.
+
+<details markdown="1">
+<summary>Output files</summary>
+
+- `vsearch_cluster/`
+  - `ASV_post_clustering_filtered.fna`: Centroid fasta file.
+  - `ASV_post_clustering_filtered.stats.tsv`: Stats table.
+  - `ASV_post_clustering_filtered.table.tsv`: ASV table.
+
+
+</details>
+
```

Comment:
I propose to move it up to filters, i.e. at the end of https://nf-co.re/ampliseq/dev/docs/output#optional-asv-filtering?
This represents its position in the chain of processes better imho.